src/topology/subset_properties.lean

```diff
@@ -789,18 +803,108 @@ end
 variables (α) [sigma_compact_space α]
 open sigma_compact_space
 
-/-- An arbitrary compact covering of a σ-compact space. -/
+/-- A choice of compact covering for a σ-compact space, chosen to be monotone. -/
 def compact_covering : ℕ → set α :=
-classical.some exists_compact_covering
+accumulate exists_compact_covering.some
```

Comment:
Do you need the monotonicity of the general compact covering, now that you have a separate construction of compact exhaustion in the locally compact case?  You could revert the edits here and on the next two lemmas.  But on the other hand, it does no harm and might be useful at some point, so if you want to leave it, that's also fine with me.

Reply:
I would prefer to leave `accumulate` here because the new definition is at least as useful as the old one.